docs/developer/kiosk.md

```diff
@@ -0,0 +1,106 @@
+
+## Kiosk controls 
+
+The Kiosk is added to improve the "console style" experience and the intention is that users will have a keyboard, connect their computer to a shoebox controller, or build themselves an arcade cabinet. You can do any of those projects with our help here: 
+
+* [Making an arcade cabinet](https://arcade.makecode.com/hardware/raspberry-pi/wooden-cabinet)
+* [Making a shoe box controller](https://arcade.makecode.com/hardware/shoebox-controller)
+
+There's a full list of do it yourself projects at:
+
+* [Do it yourself hardware](diy-hardware)
+
+Because of this, the only way to interact with the kiosk is through keyboard controls. 
+
+![Keyboard controls](/static/developer/kiosk/keyboard-controls.png)
+
+Two additional keyboard controls are:
+
+* **Spacebar**: Select
+* **ESC**: Exit
+
+## Adding a game to the Kiosk 
+
+Use the up arrow or trigger your joystick in the upwards direction to select the **Add your game** button.
+
+![Starting screen](/static/developer/kiosk/select-a-game.png)
+
+Use the space bar or the a button to select the button. Then, scan the QR code on the Kiosk.
+
+![Add your game screen](/static/developer/kiosk/add-your-game.png)
+
+Get the share link or QR code of the game you created in MakeCode Arcade.
+
+![Share project window](/static/developer/kiosk/share-project.png)
+
+The steps to get your game's share link or QR code are outlined [below](#share-link-qr-code).
+
+Use the page that looks like the one below to add your game.
+
+![Add the game to a kiosk](/static/developer/kiosk/add-to-kiosk.png)
+
+If you submit/scan your game successfully, your game will be launched on the Kiosk. If you don't see your game, wait 5 seconds as the Kiosk might be polling for it. If you do not see your game in a couple of seconds, there might be something wrong. Check the logs on the page using the Developer Tools window.
+
+## Removing all user uploaded games from the Kiosk 
+
+At some point you may decide that you no longer want your games in the Kiosk anymore. If you want to get rid of all the user-uploaded games from Kiosk, follow these steps:
+
+1. Inspect the page by right clicking on the Kiosk's landing page and selecting 'Inspect'.
+This will bring up the Developer Tools window. 
+2. In the 'Application' tab, find the 'Storage' section.
+3. Expand 'Local Storage'.
+4. Click https://arcade.makecode.com to see all the items.
+5. Filter the items with the word 'user'.
+6. Click on the entry 'UserAddedGames'.
+7. Press backspace or delete on your keyboard.
+8. Close the Developer tools.
+9. Refresh the kiosk. You will now have the default kiosk.
+
+## Getting a game's share link or QR code #share-link-qr-code
+
+You can get a game's share link or QR code using these steps:
+
+1. Click on the three connected dots in the upper right corner of the Arcade editor.
+2. Make sure to change your project title to something fun and update the project thumbnail if desired. Click 'Share Project'.
```

Comment:
<img width="708" alt="image" src="https://user-images.githubusercontent.com/49178322/213319136-9af623a2-226d-4a98-abb3-11fa53638a89.png">

Reply:
The ordered list will lose its numbering order (CSS glitch) once paragraphs are inserted (added images). So, the steps are just a para sequence.